### PR TITLE
twolame: init at 2017-09-27

### PR DESCRIPTION
**Pull request by AndersonTorres**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/libraries/twolame/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  meta = with stdenv.lib;{
+    description = "A MP2 encoder";
+    longDescription = ''
+      TwoLAME is an optimised MPEG Audio Layer 2 (MP2) encoder based on
+      tooLAME by Mike Cheng, which in turn is based upon the ISO dist10
+      code and portions of LAME.
+    '';
+    homepage = http://www.twolame.org/;
+    license = with licenses; [ lgpl2Plus ];
+  };
```
Comment (orivej) [inline pkgs/development/libraries/twolame/default.nix]: Please add platforms and maintainers.

**Review by orivej — COMMENTED** [review]
Later this should be tested on Darwin and added to ffmpeg-full, but that is not necessary to merge this. (CC @codyopel.)

File: pkgs/development/libraries/twolame/default.nix (hunk 1 above)
Comment (AndersonTorres) [reply pkgs/development/libraries/twolame/default.nix]: I will put "unix", but I don't have a Darwin here to test it.

Comment (vyp) [reply pkgs/development/libraries/twolame/default.nix]: Maintainers?

Comment (AndersonTorres) [reply pkgs/development/libraries/twolame/default.nix]: @vyp done!

File: pkgs/development/libraries/twolame/default.nix [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub
+, autoreconfHook, pkgconfig, libsndfile }:
+
+stdenv.mkDerivation rec {
+
+  name = "twolame-${version}";
+  version = "git-2017-09-27";
```
Comment (orivej) [inline pkgs/development/libraries/twolame/default.nix]: https://nixos.org/nixpkgs/manual/#sec-package-naming

> The version part of the name attribute must start with a digit (following a dash) — e.g., "hello-0.3.1rc2".
> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".

File: pkgs/development/libraries/twolame/default.nix [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+  src = fetchFromGitHub {
+    owner = "njh";
+    repo = "twolame";
+    rev = "977c8ac55d8ca6d5f35d1d413a119dac2b3b0333";
+    sha256 = "1rq3yc8ygzdqid9zk6pixmm4w9sk2vrlx217lhn5bjaglv7iyf7x";
+  };
+
+  buildInputs = [ autoreconfHook pkgconfig libsndfile ];
```
Comment (orivej) [inline pkgs/development/libraries/twolame/default.nix]: Please move `autoreconfHook` and `pkgconfig` to `nativeBuildInputs`.

File: pkgs/development/libraries/twolame/default.nix (hunk 2 above)
Comment (orivej) [reply pkgs/development/libraries/twolame/default.nix]: But don't append unstable, since this is the only version and you treat it as stable.

Comment (AndersonTorres) [reply pkgs/development/libraries/twolame/default.nix]: In sum: erasing the `git-` part?

Comment (orivej) [reply pkgs/development/libraries/twolame/default.nix]: Yes. (And from the commit message too.)

**Comment by orivej** [comment]
Thank you!

**Pull request merged**